Public/styles/more-info.css

```diff
@@ -15,4 +15,5 @@
 .more-info a:focus {
   color: #f5f5f5;
   font-weight: 400;
+  text-decoration: underline;
```

Comment:
not needed since this is default behaviour (and is not overwritten for :focus) 😊 !